### ISSUE TITLE
([CATA][Quests][Sholazar Basin] Oracles Reputation quests

### ISSUE THREAD

**Issue by Talonnrgwow** [NRG-Wow/NRGWow-Bugtracker#163]
Reporter Name: Talon

Description: all the Oracles factions quests doesn't work when you have to contact the opposite Frenzyheart faction by killing them for example.

iam linking the quests i had issues with:
https://www.wowhead.com/quest=12705/will-of-the-titans
https://www.wowhead.com/quest=12575/the-lost-mistwhisper-treasure
https://www.wowhead.com/quest=12762/power-of-the-great-ones
https://www.wowhead.com/quest=12761/mastery-of-the-crystals
https://www.wowhead.com/quest=12576/forced-hand

While the target's are marked as red, i cant attack or contact to them, it say's "unavailable target".
It would be good for example to make them autocomplete, or to allow us to kill or contact to the mentioned targets in the linked quests!

**Comment by Cannonbow** [comment]
Yeah agree with you happened to me too also there are some quests after those that we cannot due like to rescued some oracles npc some quests are bugged on Sholazar but rep is obtainable by daily quests and into the basin achiev is obtainable too
